controls/roles/validator-import/templates/service.yaml

```diff
@@ -0,0 +1 @@
+{{ stereum.manage_service.configuration | to_nice_yaml(indent=2) }}
```

Comment:
I think that's copy&paste, no purpose right?

Reply:
yes, was just copy&paste and I'll delete it